wknml/nml_utils.py

```diff
@@ -29,13 +29,13 @@ def calculate_angle_between_vectors(vector1: np.ndarray, vector2: np.ndarray) ->
 
 def calculate_distance_between_nodes(node1: Dict, node2: Dict, scale: np.ndarray) -> float:
     difference_vector = get_vector_between_nodes(node1, node2, scale)
-    return np.sqrt(difference_vector.dot(difference_vector))
+    return vector_length(difference_vector)
 
 
 def get_padding_node_position(node1: Dict, node2: Dict, relative_distance_along_vector: float, scale: np.ndarray) -> List[int]:
```

Comment:
Can you add a comment for this function? It's not entirely clear to me what the inputs/output means :see_no_evil:

Reply:
I think, you added a comment for `calculate_distance_between_nodes` but I was referring to `get_padding_node_position` :)

Reply:
Eh yea sry :see_no_evil: